app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -803,22 +803,25 @@ class AdvancedFilter extends React.Component {
   renderSelectStatement = (filter, index, value) => {
     return (
       <React.Fragment>
-        <Form.Control
-          as="select"
-          value={value}
-          className="advanced-filter-select py-0 my-0"
-          aria-label="Advanced Filter Option Select"
-          onChange={event => {
-            this.changeValue(index, event.target.value);
-          }}>
-          {filter.options.map((option, op_index) => {
-            return (
-              <option key={index + 'opkeyop-f' + op_index} value={option}>
-                {option}
-              </option>
-            );
-          })}
-        </Form.Control>
+        <div style={{ display: 'flex' }}>
```

Comment:
There's an issue here where the tooltip icon is not right aligned.  My guess is you don't need this div but haven't had a chance to play with it

![Screen Shot 2021-05-21 at 11 53 17 AM](https://user-images.githubusercontent.com/35042815/119166285-70479980-ba2c-11eb-943d-d0e1342092be.png)